lib/index.js

```diff
@@ -451,6 +451,37 @@ Flightplan.prototype.availableTargets = function() {
   return Object.keys(this._targets);
 };
 
+/** 
+ * Convenience method to allow easier access to runtime options
+ *
+ * ```javascript
+ * plan.options(); // get all options
+ * plan.options('myOption'); // get one specific option
+ * ```
+ *
+ * @method options(key)
+ */
+Flightplan.prototype.options = function(key) {
+  if(typeof key === 'undefined') {
+    return this.runtime.options;
+  }
+  return this.runtime.options && this.runtime.options[key];
+};
+
+/** 
+ * Convenience method to allow easier access to runtime hosts
+ *
+ * ```javascript
+ * plan.hosts();
+ * plan.hosts(0);
+ * ```
+ *
+ * @method hosts([index])
+ */
+Flightplan.prototype.hosts = function(index) {
+  return typeof index === 'number' ? this.runtime.hosts[index] : this.runtime.hosts;
```

Comment:
Same as above, there's no need to check whether this is a number or another type.